kubernetes/windows/setup.ps1

```diff
@@ -67,9 +67,12 @@ Write-Host ('Extracting Certificate Generator Package')
     Expand-Archive -Path /opt/amalogswindows/certificategenerator.zip -Destination /opt/amalogswindows/certgenerator/ -Force
 Write-Host ('Finished Extracting Certificate Generator Package')
 
-Write-Host ('Installing Windows Azure Monitor Agent');
+$windowsazuremonitoragent = [System.Environment]::GetEnvironmentVariable('WINDOWS_AMA_URL')
+if ([string]::IsNullOrEmpty($windowsazuremonitoragent)) {
+    $windowsazuremonitoragent = "https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/genevamonitoringagent.46.3.2.zip"
```

Comment:
We should add log message about this default so that we can troubleshoot issues if any.

Reply:
If we specify the windowsazuremonitoragent  in the pipeline variable, then we don't need to hardcode here right? And we should also log it in build pipeline

Reply:
Not defaulting in the pipeline so that we have a log history of when we made the changes for default version. We do print the url link log in the next line